src/pages/agile/chickens-versus-pigs/index.md

```diff
@@ -4,21 +4,17 @@ title: Chickens Versus Pigs
 ## Chickens Versus Pigs
 
 "Chickens versus Pigs" refers to a story about a chicken and a pig, where the chicken proposes they open a restaurant called "Ham-n-Eggs".
-The pig refuses, because while the chicken just needs to lay eggs, the pig has more at stake.
+The pig refuses, because while the chicken just needs to lay eggs, the pig has more at stake. **The Chicken is involved, but the Pig is committed!**
```

Comment:
I'm not sure this concept requires an explanation! Unless people generally don't know that ham comes from pigs?

Reply:
Hi JP, 

I had this a few times that people did not dig the point on the first glance. And since the introduction of FCC said: write it in plain and easy English, I tried. :-)